vanilla-installer/gui.py

```diff
@@ -22,185 +32,234 @@
 parser.add_argument("--safegui", type=bool)
 parser.add_argument("--litegui", type=bool)
```

Comment:
These are unused now, the GUI isn't meant to be invoked via a CLI anyway since we have a dedicated CLI.